src/ifp/test/init_floorplan1.py

```diff
@@ -0,0 +1,27 @@
+from openroad import *
+import odb
+
+def make_rect(design, xl, yl, xh, yh):
+    xl = design.micronToDBU(xl)
+    yl = design.micronToDBU(yl)
+    xh = design.micronToDBU(xh)
+    yh = design.micronToDBU(yh)
+    return odb.Rect(xl, yl, xh, yh)
+
+tech = Tech()
+tech.readLEF("Nangate45/Nangate45_tech.lef", "tech")
```

Comment:
Do tech and stdcells have any special meaning, or are they just opaque labels?

Reply:
dbLib::create takes a name.  I don't think we use it for much but that its purpose.  When we read the tech lef we don't strictly need a name if there are no stdcells in that file I don't think we need a separate api.

Reply:
May I suggest an override to this API then.

If we commonly pass tech and stdcell through eventually someone is going to rely on that, and that's just not a very fun API to work with. I think it would be good to offer an overload with enum and string. That way we can keep the labels we use commonly stable.

so something like: Which would be translated into strings.

tech.readLEF("Nangate45...", LefType.TECH)
tech.readLEF("Nangate45...", LefType.STANDARD_CELLS)
tech.readLEF("Nangate45...", "arbitrary_label")

Reply:
In read_lef we just synthesize the name from the .lef filename.  I can do that if it is preferrable to passing one in.  Having multiple APIs just puts the burden on the user to get it right whereas read_lef auto detects.

Reply:
Yeah I think that's preferable